pilot/pkg/xds/ads_test.go

```diff
@@ -92,6 +93,10 @@ func TestAgent(t *testing.T) {
 	}
 
 	t.Run("agentProxy", func(t *testing.T) {
+		credFetcher, err := credentialfetcher.NewCredFetcher(security.K8S, "", "")
```

Comment:
Same - this must be opt in, not required.

Reply:
See my comments above.